components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/welcome-page/first-launch-welcome.js

```diff
@@ -0,0 +1,207 @@
+
+                var bodyDiv = $('<div></div>');
+                var newButton = $('<button></button>');
+                var openButton = $('<button></button>');
+                var buttonGroup1 = $('<div></div>');                
+
+                var bodyTitleSpan = $('<span></span>');
+                var samplesDiv = $('<div></div>');
+
+                backgroundDiv.addClass(_.get(this._options, 'cssClass.parent'));
+                mainWelcomeDiv.addClass(_.get(this._options, 'cssClass.outer'));
+                headingDiv.addClass(_.get(this._options, 'cssClass.heading'));
+                headingTitleSpan.addClass(_.get(this._options, 'cssClass.headingTitle'));
+                headingTitleSpan2.addClass(_.get(this._options, 'cssClass.headingTitle'));
+                newButton.addClass(_.get(this._options, 'cssClass.buttonNew'));
+                openButton.addClass(_.get(this._options, 'cssClass.buttonOpen'));
+                headingGroup1.addClass(_.get(this._options, 'cssClass.headingTop'));
+                buttonGroup1.addClass(_.get(this._options, 'cssClass.btnWrap1'));
+                
+
+                bodyDiv.addClass(_.get(this._options, 'cssClass.body'));
+                bodyTitleSpan.addClass(_.get(this._options, 'cssClass.bodyTitle'));
+                samplesDiv.addClass(_.get(this._options, 'cssClass.samples'));
+                samplesDiv.attr('id', 'samplePanel');
+
+                newButton.text("New");
+                openButton.text("Open");
+
+                headingTitleSpan.text("Welcome to");
+                headingTitleSpan2.text("Data Analytics Composer");
+//                bodyTitleSpan.text("Try out our samples / Templates");
+
+                wrapTitle.append(headingTitleSpan);
+                wrapTitle.append(headingTitleSpan2);
+                headingGroup1.append(wrapTitle);
+
+                buttonGroup1.append(newButton);
+                buttonGroup1.append(openButton);
+
+                headingDiv.append(headingGroup1);
+                headingDiv.append(buttonGroup1);                
+
+                bodyDiv.append(bodyTitleSpan);
+                bodyDiv.append(samplesDiv);
+
+                mainWelcomeDiv.append(headingDiv);
+                mainWelcomeDiv.append(bodyDiv);
+                backgroundDiv.append(mainWelcomeDiv);
+
+                this._$parent_el.append(backgroundDiv);
+                this.$el = backgroundDiv;
+
+                var innerDiv = $('<div></div>');
+                innerDiv.attr('id', "innerSamples");
+                samplesDiv.append(innerDiv);
+
+                var command = this._options.application.commandManager;
+                var browserStorage = this._options.application.browserStorage;
+
+                var samples = _.get(this._options, "samples", []);
+                
+                var config;
+                var servicePreview;
+
+                var self = this;
+
+//                for (var i = 0; i < samples.length; i++) {
```

Comment:
Yes we need that to add further improvements to welcome page. Which are not yet properly implemented.For this release we are providing limited functionality in welcome page